evap/staff/tools.py

```diff
@@ -7,11 +8,40 @@
 from django.core.cache import cache
 from django.core.cache.utils import make_template_fragment_key
 from django.db import transaction
+from django.conf import settings
 
 from evap.evaluation.models import UserProfile, Course, Contribution
 from evap.grades.models import GradeDocument
 
 
+def generate_import_file_name(user_id):
+    return settings.MEDIA_ROOT + '/temp_import_files/' + user_id + '.xls'
+
+
+def save_import_file(excel_file, user_id):
+    filename = generate_import_file_name(user_id)
+    os.makedirs(os.path.dirname(filename), exist_ok=True)
+    with open(filename, "wb") as file:
+        for chunk in excel_file.chunks():
+            file.write(chunk)
+    excel_file.seek(0)
+
+
+def delete_import_file(user_id):
+    filename = generate_import_file_name(user_id)
+    if not os.path.isfile(filename):
+        return False
+    os.remove(filename)
+
+
+def get_import_file_name_if_exists(user_id):
```

Comment:
please check whether returning the actual file (not name) or None simplifies things